src/shared/constants.h

```diff
@@ -228,3 +228,5 @@ static const unsigned NUM_GENEVA_ANGLES = 5;
 
 // Jetson Nano Constants
 static const double MAX_JETSON_TEMP_C = 97;
+
+static const double RTT_S = 0.03;
```

Comment:
nit: this can be `constexpr` which will be replaced directly during compilation and gives us a tiny performance benefit